### PR TITLE
Update links and tools in overview

### PR DESCRIPTION
**Pull request by dcwalk**
Waiting on this to fit in with @ebarry updates

### CODE REVIEW

**Comment by dcwalk** [comment]
Big restructure @titaniumbones this gets us in sync with the NYC event 😁

File: README.md [hunk 1]
```diff
@@ -10,110 +10,73 @@ If you are running your own event you've come to the right repo. The documentati
...
-## Review of Our Tools
-
-| Tool Name | Description | Status | Language |
-|-----------|-------------|--------|----------|
+| Current Tools | Description | Status | Language |
+|---------------|-------------|--------|----------|
 | [**Data Rescue Workflow docs**](https://github.com/datarefugephilly/workflow) | Detailed descriptions of the phases of event workflow | **Working** | -- |
-| [**Presidential Harvest Nomination Tool**](https://github.com/edgi-govdata-archiving/presidential-harvest-nomination-tool) | A Chrome extension to simplify the nomination process at archiv-a-thons | **Working** | HTML & JavaScript |
+| [**Nomination Tool**](https://github.com/edgi-govdata-archiving/eot-nomination-tool) | Chrome extension to simplify the nomination process at archiv-a-thons | **Working** | HTML & JavaScript |
+| [**Harvesting Tool**](https://github.com/edgi-govdata-archiving/harvesting-tools) | A collection of code snippets designed to be dropped into the data harvesting process directly after generating the zip starter kit | **Working** | Various |
```
Comment (titaniumbones) [inline README.md]: Harvesting Tools (plural)

File: README.md [hunk 2]
```diff
@@ -10,110 +10,73 @@ If you are running your own event you've come to the right repo. The documentati
...
+| [**Zip Starter**](https://github.com/edgi-govdata-archiving/zip-starter) | Automate zip folder creation for Harvesting during pipeline | **Working** | Go |
+| **Pipeline App** |  | **Working** | Meteor, JavaScript |
```
Comment (titaniumbones) [inline README.md]: add link to URL of app? or "If interested, ask @b5"?

**Review by titaniumbones — COMMENTED** [review]
2 small changes, but **SWEET** much better! decide on my comments then squash/merge yourself? 

going to bed. You ***ROCK*** 🚀 🤓 !

File: README.md [hunk 3]
```diff
@@ -10,110 +10,73 @@ If you are running your own event you've come to the right repo. The documentati
...
+
+## Projects
+
+Projects are tracked via issues in their own repos and managed through our [Project Tracking Board](https://github.com/edgi-govdata-archiving/overview/projects/2).
+
+### Monitoring Websites
+
+| Tool Name | Description | Status |
+|-----------|-------------|--------|
+| [**Page Freezer Tools**](https://github.com/edgi-govdata-archiving/pagefreezer-tools) | Initial Repo for a Page Freezer filtering toolkit  | **In Progress** |
```
Comment (titaniumbones) [inline README.md]: parefreezer tools is deprecated, I'd forgotten that I created it. WIll go now and destroy. remove?

**Comment by dcwalk** [comment]
Thanks for the review, made changes and merging!

**Pull request merged**